src/ui/controls/logsdialog.cpp

```diff
@@ -0,0 +1,17 @@
+#include "logsdialog.hpp"
+
+using namespace NickvisionTubeConverter::UI::Controls;
+
+LogsDialog::LogsDialog(GtkWindow* parent, const std::string& title, const std::string& logs, const std::string& cancelText, const std::string& destructiveText, const std::string& suggestedText) : MessageDialog(parent, title, "", cancelText, destructiveText, suggestedText)
+{
+    //Logs
+    m_scrolledWindow = gtk_scrolled_window_new();
+    gtk_widget_set_size_request(m_scrolledWindow, 500, 500);
+    m_textView = gtk_text_view_new();
+    m_textBuffer = gtk_text_view_get_buffer(GTK_TEXT_VIEW(m_textView));
```

Comment:
Could you also try adding `gtk_style_context_add_class(gtk_widget_get_style_context(m_textView), "card");` and see how the text view looks rounded?